packages/database/src/migrations/20210512004319-AddPSSSConfirmedDataPerSyndromeReport-modifies-data.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+import { generateId, insertObject, arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const SYNDROMES = ['AFR', 'DIA', 'ILI', 'PF', 'DLI'];
+
+const getReportCode = syndrome => `PSSS_${syndrome}_Confirmed_Report`;
+
+const getTotalConfirmedCasesIndicator = syndrome => `PSSS_Accumulated_${syndrome}_Confirmed_Cases`;
+
+const getReport = (syndrome, permissionGroupId) => {
+  const reportCode = getReportCode(syndrome);
+  const totalConfirmedCasesIndicator = getTotalConfirmedCasesIndicator(syndrome);
+
+  return {
+    id: generateId(),
+    code: reportCode,
+    config: {
+      fetch: {
+        dataElements: [totalConfirmedCasesIndicator],
+      },
+      transform: [
+        'keyValueByDataElementName',
+        'mostRecentValuePerOrgUnit',
+        {
+          // change key names
+          transform: 'select',
+          "'totalCases'": `$row.${totalConfirmedCasesIndicator}`,
+        },
+      ],
+    },
+    permission_group_id: permissionGroupId,
+  };
```

Comment:
So it was decided that we won't support a `# sites reported` accumulated value? It was gonna be tricky to do so I'm somewhat glad (but I was kinda curious to see if we could pull it off 😉)

Reply:
Yeap Erin has confirmed this!